webdriver/tests/bidi/script/call_function/internal_id.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from ... import recursive_compare, any_string
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "return_structure, result_type",
```

Comment:
`result_type` can be removed from here now

Reply:
but it's still used to build the value here: https://github.com/web-platform-tests/wpt/pull/36041/files#diff-609470bfe4f5bba1b3da43624062982a560723dbd8cf36af1b27cdab9a34194fR31